modules/mod_random_image/src/Helper/RandomImageHelper.php

```diff
@@ -103,9 +110,13 @@ public static function getImages(&$params, $folder)
             $i = 0;
 
             foreach ($files as $img) {
-                if (!is_dir($dir . '/' . $img) && preg_match('/' . $type . '/', $img)) {
-                    $images[$i] = new \stdClass();
+                if (is_dir($dir . '/' . $img)) {
+                    continue;
+                }
 
+                $ext = pathinfo($img, PATHINFO_EXTENSION);
+                if (\in_array(strtolower($ext), $extensions, true)) {
```

Comment:
```suggestion
                $ext = pathinfo($img, PATHINFO_EXTENSION);

                if (\in_array(strtolower($ext), $extensions, true)) {
```
CS fix - empty line before control structures.